CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+
+* The documentation is module-based, which means most sections are individual modules. These modules are available in the */docs/topics/modules/* folder.
+* The modules are linked together to form a chapter, document, or guide using assemblies. These assemblies are asciidoc files in the */docs/topics/* folder. For example, the */docs/topics/getting-started-guide.adoc* file is the main assembly for the Getting Started Guide and links to other, smaller assemblies for parts or chapters within the guide.
+* Module files naming conventions closely follow the title of the corresponding section. For example, a section in the Getting Started document with the title *Working with pipelines* is named **working_with_pipelines.adoc** in the *modules* folder. In some cases, this is not a direct match, so to confirm the name of the asciidoc file for a section, click the section title and the ID after the *#* in the URL is the name of the module file. For example, a URL such as https://docs.openshift.io/getting-started-guide.html#viewing_build_pipeline_oso has a slightly different title but the ID after the hash symbol (**viewing_build_pipeline_oso**) is the name of the file.
+
+## Conventions and style guidelines
+
+To get started quickly, we recommend reviewing the following conventions that are relevant for this documentation suite:
+
+**Writing conventions**
+
+* For instructions, state the location, then action and subject. For example:
+  * Incorrect: *Click OK in the XYZ tab to continue.*
+  * Correct: *In the XYZ tab, click OK to continue.*
+  * Use gerunds in titles when possible. For example:
+    * Incorrect: *1.1 Create a new project*
+    * Correct: *1.1 Creating a new project*
+* All titles should be in sentence case. Exceptions for words that are proper nouns. For example:
+  * Incorrect: *1.2 Configuring Your Che Workspace*
+  * Correct: *1.2 Configuring your Che workspace*
+* URLs must be bared in the text, not hidden. For example:
+  * Incorrect: *Navigate to the [Red Hat website](www.redhat.com).* 
+  * Correct: *Navigate to (www.redhat.com).*
+  
+**Markup conventions**  
+
+* TBD 
+  
+For comprehensive references for documentation conventions, see the following:
+
+  * The IBM Style Guide. A subset of this guide is available [here](https://www.ibm.com/developerworks/library/styleguidelines/).
+  * The [Red Hat Asciidoc Markup conventions](https://redhat-documentation.github.io/asciidoc-markup-conventions/).
+  <!-- * The [CCS Documentation Conventions][url here when available] UNCOMMENT THIS WHEN READY-->
+
+## Editing the documentation
+
+To edit the documentation:
+
+1. Ensure that an issue with the details of the problem are logged at https://github.com/fabric8io/fabric8-online-docs/issues for context and assign it to yourself.
+
+2. Clone the documentation repository and make the relevant changes in a branch named after the issue you are working on.
+
```

Comment:
How about putting these steps right at the start. Put in the conventions section next and link it in this step just as you have done in the 3rd step. So the 3 sections in Contributing would be 1. steps, 2. conventions, 3. build info.

Reply:
Sure, I'll try that and see what we all think

Reply:
@mishaone I believe step #2 should be 'Fork the repo to your Github space' followed by cloning that new fork of the repo.

Reply:
@mishaone Actually, https://guides.github.com/introduction/flow/ I see the GH flow doesn't use forks. Hmmm. Maybe better link to https://gist.github.com/Chaser324/ce0505fbed06b947d962

Reply:
Do we want more detailed instructions for Git here? I was thinking more generic, something like clone the repo, make the changes/additions, create a PR. This assumes the people making the change know enough that we don't need to give specific commands. However, if there is a reason to, I'm happy to add detailed commands. WDYT, @Preeticp and @rkratky

Reply:
@mishaone +1 to not including actual commands.